aidants_connect_web/migrations/0003_auto_20190723_1508.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.3 on 2019-07-23 13:08
+
+from django.db import migrations, models
```

Comment:
⚠️ Don't forget to rebase on master and replay the migrations

Reply:
This branch seems up to date to me, what am I missing?

Reply:
I might be a bit paranoid about the migrations, but if you are sure everything is up to date with master, then you can merge :)

Reply:
Made some cleaning, it seemed that some migration files were left although they were squashed in another file. Is it good with you?

Reply:
The squashed files took the files from migration `0001` to `0003` and squashed them in `0001_squashed_0003_auto_20190725_1048.py`
The squashed files should be kept until `integ` has been updated.

Then, we can delete the squashed migration files.

1. before merging, I think you should update `integ` with the latest master
2. I think you can discard `0001_initial.py` along with the other squashed migrations